docs/components/radio.md

```diff
@@ -106,6 +160,36 @@ Any valid attribute will be added to the input. You can read more about `$attrs`
 </KRadio>
 ```
 
+-  `before` - Available only when `type` prop is `card`. Will place content above the title and description (if provided).
+-  `after` - Available only when `type` prop is `card`. Will place content below the title and description (if provided).
```

Comment:
suggestion: Can we rename these slots? You'll need to update the usage as well. I think it makes sense to name the slots accordingly since they only correspond to the specific `type`

`cardTop` and `cardBottom`

Reply:
How about `header` and `footer` @adamdehaven?

Reply:
I saw your comment earlier about how you can’t use the single slot because of the `description` prop but I’m still not sure I understand how that prevents the single slot solution?

If the `type` is `card` it could essentially just “override” all of the other slots and use the single-slot solution you were originally going with along with the new classes. Would this work?

Reply:
You know what - I haven't tried that tbh. I was trying to hit two birds with one stone and make the component flexible so that in both default radio input and card they could choose between using props or slots for label - but that approach adds too many complications. "Overriding" the default slot when `type` is `card` like you suggested is gonna be much cleaner solution - only potential downside is that they will only be able to use default slot, but not the `label` or `description` props, is that okay?

Reply:
Yes I think that's fine as long as you document that only the default slot is used in the description of the `type` prop when it is set to `card`